angular-frontend/AUTHORIZATION.md

```diff
@@ -0,0 +1,81 @@
+# Authorization
+To leverage the benefits of Tanzu AppSSO Authserver, there are a few things you should take into consideration.
+
+## Inner Loop (Development Mode)
+### Configuration of auth.conf.json
+Your Angular app should have an `src/assets/auth.conf.json` file that needs configuration. The items in the file correspond to the following:
+
+```json
+{
+  "clientId": "your-client-id",
+  "authority": "https://your-authorization-server.com/",
+  "scope": [
+    "openid" // This list MUST contain openid
+  ]
+}
+```
+
+__Note:__ The scpoes listed here are default scopes. Replace with the scopes appropriate for your application. Any scopes you 
+want to have available here also need to be listed in the `clientregistration.yaml`
+
+## Authserver Configuration
+To enable a single page app with an AppSSO Authserver, you will need to implement CORS. You can specify CORS settings in the AuthServer CR spec.cors section. Here are some examples:
+
+```yaml
+kind: AuthServer
+spec:
+  cors:
+    allowOrigins:
+      - http://127.0.0.1:4200/
+```
+
+__Note:__ Replace the redirect uri with your application's uri
+
+Alternatively, to allow all origins, the AuthServer CR should look like this:
+
+```yaml
+kind: AuthServer
+metadata:
+  annotations:
+    - "sso.apps.tanzu.vmware.com/allow-unsafe-cors": ""
+spec:
+  cors:
+    allowAllOrigins: true
+```
+
+Note that localhost redirect URIs are not allowed with AppSSO. For local testing, it is recommended to use npm run serve, which includes the --host 127.0.0.1 flag. The AuthServer should recognize your client, so it is necessary to define your ClientRegistration CR as follows:
```

Comment:
**[suggestion]**: npm run serve should be backticked, as well as --host 127.0.0.1 flag

**[suggestion]**: ClientRegistration should be in back ticks